corehq/apps/es/tests/test_case_search_es.py

```diff
@@ -253,26 +253,27 @@ def tearDown(self):
         ensure_index_deleted(CASE_SEARCH_INDEX)
         super(TestCaseSearchLookups, self).tearDown()
 
-    def _make_case(self, domain, case_properties):
+    def _make_case(self, domain, case_properties, index=None):
         # make a case
         case_properties = case_properties or {}
         case_id = case_properties.pop('_id')
         case_name = 'case-name-{}'.format(uuid.uuid4().hex)
         owner_id = case_properties.pop('owner_id', None)
         case = create_and_save_a_case(
-            domain, case_id, case_name, case_properties, owner_id=owner_id, case_type=self.case_type)
+            domain, case_id, case_name, case_properties, owner_id=owner_id, case_type=self.case_type, index=index)
         return case
 
-    def _bootstrap_cases_in_es_for_domain(self, domain):
+    def _bootstrap_cases_in_es_for_domain(self, domain, input_cases):
```

Comment:
This is a nice util - might pull it out and use elsewhere.  I recently did [the same thing](https://github.com/dimagi/commcare-hq/blob/dc87878ccb98acbee5a326b5f351024323eb5d15/corehq/apps/hqcase/tests/test_serialization.py#L25-L25) and saw this, but needed support for indices so I rolled my own.

Reply:
I ended up re-implementing this [here](https://github.com/dimagi/commcare-hq/pull/29609/files#diff-b83ca8df28e8aaf7279428ab7c8861172f10a828da546c831bc77cd76bfdb1eb), happy to hear feedback.